src/main/java/com/aws/greengrass/clientdevices/auth/connectivity/ConnectivityInformation.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.clientdevices.auth.connectivity;
+
+import software.amazon.awssdk.services.greengrassv2data.model.ConnectivityInfo;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class ConnectivityInformation {
```

Comment:
As far as I understand this this stores some Connection Information but more specifically the HostAddressInformation. I wonder if it would make more sense to call this HostAddressInformation and have the aggregate class be called ConnectivityInformation